run.sh

```diff
@@ -36,16 +36,59 @@ function output() {
   fi
 }
 
+function do_ftrace() {
+  if [[ -z "${ftraceMode}" ]]; then
+    return
+  fi
+
+  d=$(date '+%Y-%m-%dT%H:%M:%S')
```

Comment:
add Z to the end and it will be possible to cut and paste as a parsable timestamp

Reply:
Done (and moved to hook.sh)